AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -1564,7 +1564,7 @@ swu.de##.js-cookiebanner-wrap
 netmovies.com.br##div[class^="styles_acceptCookiesWrapper__"]
 hoyer.de###app > div[class^="style_backdrop__"]
 passeidireto.com##pd-cookie-banner
-tonspion.de,bier.by,test.de,kicker.de##div[id^="cmpbox"]
+weser-kurier.de,tonspion.de,bier.by,test.de,kicker.de##div[id^="cmpbox"]
```

Comment:
The scroll page is blocked, the site needs other rules.
Will fix this directly.